Example/ViewController+StyleManager.swift

```diff
@@ -4,12 +4,18 @@ import MapboxNavigation
 import MapboxMaps
 
 extension ViewController : StyleManagerDelegate {
+    
     func location(for styleManager: MapboxNavigation.StyleManager) -> CLLocation? {
+        // Sometimes when style changes user might be already actively navigating, this means that
+        // `navigationMapView` is no longer available.
+        if navigationMapView == nil { return nil }
```

Comment:
Fixes crash in example application whenever style changes during active navigation.